src/components/WorkspaceControlPanel.js

```diff
@@ -83,25 +105,58 @@ class WorkspaceControlPanel extends Component {
         classes={{ paper: classNames(classes.drawer) }}
         open
       >
-        <Menu id="add-form" anchorEl={anchorEl} open={Boolean(anchorEl)} onClose={this.handleClose}>
-          <ConnectedManifestForm id="add-form" setLastRequested={this.setLastRequested} />
+        <Menu
+          id="ws-ctrl-pnl-mn"
+          anchorEl={anchorEl}
+          open={Boolean(anchorEl)}
+          onClose={this.handleAddManifestClose}
+        >
+          <ConnectedManifestForm
+            id="add-form"
+            setLastRequested={this.setLastRequested}
+          />
           <ul>{manifestList}</ul>
           {lastRequested}
         </Menu>
-
         <List>
-          <Fab
-            color="primary"
-            id="addBtn"
-            aria-label="Add"
-            className={classes.fab}
-            aria-owns={anchorEl ? 'add-form' : undefined}
-            aria-haspopup="true"
-            onClick={this.handleClick}
-          >
-            <AddIcon />
-          </Fab>
-          <ConnectedWorkspaceControlPanelButtons />
```

Comment:
It seems like `ConnectedWorkspaceControlPanelButtons` was removed here. But there are some changes in that component. Should we create a spin off ticket to address this later?